staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -34,6 +34,10 @@ require (
 )
 
 replace (
+	golang.org/x/exp => golang.org/x/exp v0.0.0-20190312203227-4b39c73a6495
+	golang.org/x/lint => golang.org/x/lint v0.0.0-20190409202823-959b441ac422
+	golang.org/x/net => golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
```

Comment:
net has a go 1.13 release branch, so pinning to that is ok. exp/lint/oauth2 do not, so I would expect to use the preferred versions of those

Reply:
`hack/lint-dependencies.sh  --all` will show golang.org/x/... deps that are not at their preferred level

Reply:
Ack will update